awx/main/utils/handlers.py

```diff
@@ -294,6 +294,18 @@ def __init__(self, **kwargs):
         super(AWXProxyHandler, self).__init__(**kwargs)
         self._handler = None
         self._old_kwargs = {}
+        self._auditor = logging.handlers.RotatingFileHandler(
+            filename='/var/log/tower/external.log',
+            maxBytes=1024 * 1024 * 50, # 50 MB
+            backupCount=5,
+        )
+
+        class WritableLogstashFormatter(LogstashFormatter):
+            @classmethod
+            def serialize(cls, message):
+                return json.dumps(message)
```

Comment:
Thought of this too late, but a YAML option would allow direct parsing of the file. As it stands now, it seems that we'll have to split on lines before attempting any parsing.